### PR TITLE
Minor addition to context definition

### PR DESCRIPTION
**Pull request by nacx**

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://55de5352931516bd6330-55e1d72d54648c5093cb6da556c72f45.r69.cf5.rackcdn.com/ to review your changes.

File: start/concepts.md [hunk 1]
```diff
@@ -80,3 +80,7 @@ A context represents a specific connection to a particular provider. From the pe
 Once you have created a context via the [ContextBuilder](http://javadocs.jclouds.cloudbees.net/org/jclouds/ContextBuilder.html) and are "connected" to a particular cloud service, you can either get any of the views that are supported by that provider, or go straight to the API or even to the provider level.
```
Comment (demobox) [inline start/concepts.md]: Suggestion: add it here? As follows:

"...provider level.

Creating a context is an expensive operation, so in general it is a good idea to create one context per credential and target cloud when the application starts and close it when it terminates. Contexts are thread-safe, so can be shared across your application.

It is important to [close a context](http://jclouds-javadocs.elasticbeanstalk.com/org/jclouds/Context.html) when you no longer needed, to free its associated resources."

**Comment by jclouds-commentator** [comment]
Go to http://e3fc44bd4430654115f7-165e9c5ecddbeacd39e268108e8f046e.r46.cf5.rackcdn.com/ to review your changes.

**Comment by nacx** [comment]
Pushed to [master](http://git-wip-us.apache.org/repos/asf/jclouds-site/commit/56a75bbb). Thanks!

File: start/concepts.md [hunk 2]
```diff
@@ -79,4 +79,10 @@ A context represents a specific connection to a particular provider. From the pe
 
 Once you have created a context via the [ContextBuilder](http://javadocs.jclouds.cloudbees.net/org/jclouds/ContextBuilder.html) and are "connected" to a particular cloud service, you can either get any of the views that are supported by that provider, or go straight to the API or even to the provider level.
 
+Creating a context is an expensive operation, so in general it is a good idea to create one context per credential and target cloud when the application starts and close it when it terminates. Contexts are thread-safe, so can be shared across your application.
+
+It is important to [close a context](http://jclouds-javadocs.elasticbeanstalk.com/org/jclouds/Context.html) when you no longer need it, to free its associated resources.
+
 You can also get a view back from the ContextBuilder and _later_ unwrap it to access the underlying API or provider.
+
+The context also provides access to some low level resources, such as the dependency injection framework or the executors used to perform concurrent operations, so it is important to close it once you are done, to free all its associated resources.
```
Comment (demobox) [inline start/concepts.md]: Do we still need this para?

**Comment by demobox** [comment]
> Pushed to master. Thanks!

Thanks, @nacx! See https://github.com/jclouds/jclouds-site/pull/167

**Comment by nacx** [comment]
Oh yes, I forgot to remove it. Thanks!

**Pull request closed without merging**